backend/program/settings/models.py

```diff
@@ -155,19 +157,19 @@ class OrionoidConfig(Observable):
 
 class JackettConfig(Observable):
     enabled: bool = False
-    url: str = "http://localhost:9117"
+    url: str = "http://jackett:9117" if is_elfhosted else "http://localhost:9117"
     api_key: str = ""
 
 
 class ProwlarrConfig(Observable):
     enabled: bool = False
-    url: str = "http://localhost:9696"
+    url: str = "http://prowlarr:9696" if is_elfhosted else "http://localhost:9696"
     api_key: str = ""
 
 
 class AnnatarConfig(Observable):
     enabled: bool = False
-    url: str = "http://annatar.elfhosted.com"
+    url: str = "http://elfhosted-internal.annatar" if is_elfhosted else "http://annatar.elfhosted.com"
```

Comment:
I'm still unsure about these.. think a new private repo would be best?

Reply:
ah, I guess the link doesn't resolve anyways outside of their ecosystem.. so probably not a big deal